### PR TITLE
Ddoc 596 cli updates

### PR DESCRIPTION
**Pull request by Smartoneinok**
PR for 596 - Please confirm with Alex before releasing even though he is on Pat leave!

### CODE REVIEW

File: content/guides/tooling/cli/quick-start/0-index.md [hunk 1]
```diff
@@ -26,7 +26,8 @@ This guide will take you through the following steps.
 2. [Install and configure][two] the CLI
 3. [Run commands][three] with the CLI
 4. Use [options and bulk commands][four]
-5. [Next steps][five]
+5. Explore using [Powershell scripts with the CLI][five]
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/0-index.md:29]: I miss the info about what should be explored - for example `Explore automation using Powershell scripts`

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 2]
```diff
@@ -0,0 +1,239 @@
+---
+type: quick-start
+hide_in_page_nav: true
+---
+
+# Using Powershell Scripts with the Box CLI
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:6]:
```suggestion
# Using PowerShell Scripts with the Box CLI
```

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 3]
```diff
@@ -0,0 +1,239 @@
...
+
+By itself, the CLI is already powerful, but using it alongside a Powershell
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:8]:
```suggestion
By itself, the CLI is already powerful, but using it alongside a PowerShell
```

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 4]
```diff
@@ -0,0 +1,239 @@
...
+
+### MacOS & Linux
+
+- Install [PowerShell][pwsh]
+
+If you encounter issues make sure you install both 
+[dotnet core](https://dotnet.microsoft.com/download) and 
+[PowerShell][pwsh]
+
+### Configure a Box application
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:53]:
```suggestion
### Configured Box application
```

**Review by bszwarc — CHANGES_REQUESTED** [review]
I added some comments and suggestions - of course they are just suggestions, and you decide if you want to keep them. Some of them can be applied and committed right away, and some are some structural suggestions :)

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 5]
```diff
@@ -0,0 +1,239 @@
...
+
+# Using Powershell Scripts with the Box CLI
+
+By itself, the CLI is already powerful, but using it alongside a Powershell
+script lets you complete repetitive tasks even quicker. We created an example
+[scripts][scripts] folder within our CLI GitHub repository to help
+jump start your development.
+
+For this step in the quick start, we are going to use the
+user provisioning and folder creation [script][script-1]
```
[the remark this answers is not in the document]
Comment (barduinor) [reply content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:14]: done

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 6]
```diff
@@ -0,0 +1,239 @@
...
+## Download script
+
+```bash
+git clone https://github.com/box/boxcli.git box-cli
+cd box-cli/examples/User\ Creation\ \&\ Provisioning/
+```
+
+## Modify script configuration
+
+You must adapt this script for your own environment.
```
[the remark this answers is not in the document]
Comment (barduinor) [reply content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:69]: to run in

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 7]
```diff
@@ -0,0 +1,239 @@
...
+
+### CSV Files
```
[the remark this answers is not in the document]
Comment (barduinor) [reply content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:71]: we are not actually uploading users, we are preparing the csv files. humm user list?

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 8]
```diff
@@ -0,0 +1,239 @@
...
+$script:OnboardingFolderId = box folders:upload $LocalUploadPath --id-only
+Write-Output "Uploaded local folder structre to current user's root folder 
+with $($script:OnboardingFolderId)"
+```
+
+## Run the script
+
+Now all you need to do is run the script.
+Make sure when in `powershell` you are at the base directory relative to the 
+scripts path per the configurations you specified in `Users_Create_Provision.ps1`:
```
[the remark this answers is not in the document]
Comment (barduinor) [reply content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:157]: no, it depends where did the user pointed the paths in the script

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 9]
```diff
@@ -0,0 +1,240 @@
...
+
+## Use case
+
+The use case presented in this example is to illustrate the automatic 
+creation of users:
+
+- using a `.csv` file to load users in bulk
+- have a predetermined folder structure associated to each user
+- define that folder structure using a JSON file
+- or, optionally,  create the folder structure by uploading from a local drive
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:38]:
```suggestion
- or, optionally, create the folder structure by uploading from a local drive
```
unnecessary space :)

**Review by bszwarc — COMMENTED** [review]
A couple of minor comments :)

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 10]
```diff
@@ -0,0 +1,240 @@
...
+You must adapt this script to run in your own environment.
+
+### Prepare the user list
+
+You can use the following sample files to load users:
+`Employees_1.csv`, `Employees_5.csv`, and `Employees_10.csv`. 
+Each will load 1, 5 or 10 new users.
+
+Customize these files for a test run. For example, update the 
+`Employeess_1.csv` with the following data:
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:79]:
```suggestion
`Employees_1.csv` with the following data:
```

File: content/guides/tooling/cli/quick-start/5-powershell-script-templates.md [hunk 11]
```diff
@@ -0,0 +1,240 @@
...
+
+```bash
+firstName,lastName,email
+Isaac,Newton,abc@abc.local
+```
+
+In the `Users_Create_Provision.ps1` script file, specify which `.csv` 
+file you would like to load. For example, `Employee_1.csv`:
```
Comment (bszwarc) [inline content/guides/tooling/cli/quick-start/5-powershell-script-templates.md:87]:
```suggestion
file you would like to load. For example, `Employees_1.csv`:
```

**Review by bszwarc — APPROVED** [review]

**Pull request merged**